condor_watch_q.py

```diff
@@ -432,16 +436,18 @@ def table_by_event_log(self, abbreviate_path_components=False):
                     if len(live_proc_ids) == 1:
                         x = "{}.{}".format(cluster_id, live_proc_ids[0])
                     else:
-                        x = "{}.{}-{}".format(
-                            cluster_id, min(live_proc_ids), max(live_proc_ids)
+                        x = "{}.{}".format(
+                           cluster_id, min(live_proc_ids), max(live_proc_ids)
                         )
                     live_job_ids.append(x)
 
             d[TOTAL] = sum(d.values())
             d[EVENT_LOG] = event_log_names[event_log_path]
-            d[ACTIVE_JOBS] = ", ".join(live_job_ids)
+            x = datetime.datetime.now()
```

Comment:
This is clearly wrong - we'll need to pull the submission time out of the job ads. See https://htcondor.readthedocs.io/en/latest/classad-attributes/job-classad-attributes.html for job ad attributes; it will need to be added the query projection and then pulled out of the ads and parsed.

Reply:
On further consideration, I think the approach I suggested in the previous comment is also wrong. We don't display any information that we actually acquire from the live job ads, we only do the query to find the event logs.

If we have a column like this, its information needs to be acquired from the event logs. We don't have a good architecture for doing this right now, so I'm going to say drop the SUBMITTED column for now. I'll think more about the architecture so that we can do something like this.

Even if we had this column, I don't think it's obvious what its meaning is. Is it the submit time of the last job on that row? If so, that's pretty useless in the general case unless you group by cluster id, which I don't really think is the right way to group. I think I might prefer a column for "last activity time".